docs/devel/networking.md

```diff
@@ -72,6 +72,74 @@ As of now, the existent binding mechanisms are:
 - [slirp](#slirp-binding-mechanism)
 
 ### Bridge binding mechanism
+Using the bridge `BindMechanism` requires a VMI configuration featuring a
+network whose interface type is `bridge` - the yaml file below can be used
+as reference, but please refer to the
+[user guide](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=bridge)
+for more information.
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: default
+          bridge: {}
+  networks:
+  - name: default
+    pod: {} # Stock pod network
+```
+
+Let's refer to the image below to get a better understanding of how this
+`BindMechanism` works.
+
+Bridge binding mechanism diagram
+
+![alt text](https://github.com/kubevirt/kubevirt.github.io/blob/source/assets/images/diagram.png)
+
+As can be seen in the diagram above, there are three actors at play: CNI,
+libvirt, and DHCP. For completeness sake, let's add one more actor that is
+implicit in the picture: KubeVirt.
+
+Once the bridge `BindMechanism` is triggered,
```

Comment:
BindMechanism is only used to connect VM to the Pod network. The CNI is responsible to get it to the Pod. I don't think these two should be mixed.